torch/cuda/nvtx.py

```diff
@@ -66,7 +66,7 @@ def range_end(range_id) -> None:
     _nvtx.rangeEnd(range_id)
 
 
-def device_range_start(msg, stream=0) -> int:
+def device_range_start(msg, stream=0):
```

Comment:
consider something like this?
```
def device_range_start(msg: str, stream:int) -> object:
```

Reply:
(Sorry - didn't mean to approve - was just commenting on this one line)

Reply:
Done https://github.com/pytorch/pytorch/pull/130429/commits/42266bcdf721ee217d48165740b923bfb3f9b52e